src/main/java/net/openhft/chronicle/threads/Threads.java

```diff
@@ -104,10 +107,9 @@ public static void shutdownDaemon(@NotNull ExecutorService service) {
             if (!terminated) {
                 terminated = service.awaitTermination(1, TimeUnit.SECONDS);
                 if (!terminated) {
-                    if (service instanceof ThreadPoolExecutor)
-                        warnRunningThreads(service);
-                    else
-                        Jvm.warn().on(Threads.class, "*** FAILED TO TERMINATE " + service.toString());
+                    if (!(service instanceof ThreadPoolExecutor))
+                        Jvm.warn().on(Threads.class, "*** FAILED TO TERMINATE " + service);
+                    warnRunningThreads(service);
```

Comment:
I left the FAILED TO TERMINATE error in because the message might be a bit ambiguous in the event we can't resolve any threads.